openverse_catalog/dags/providers/provider_api_scripts/flickr.py

```diff
@@ -129,6 +181,32 @@ def get_batch_data(self, response_json):
 
         if response_json is None or response_json.get("stat") != "ok":
             return None
+
+        # Detect if this batch has more than the max_unique_records.
+        detected_count = self.get_record_count_from_response(response_json)
+        if detected_count > self.max_unique_records:
+            logger.error(
+                f"{detected_count} records retrieved, but there is a"
+                f" limit of {self.max_unique_records}."
+            )
```

Comment:
This log was so useful when observing this PR in airflow. Before even reading the code it gave a great understanding of what was happening 👍